{
  throw new IllegalStateException("evaluate() shall had failed with exception");
}
